app/helpers/application_helper.rb

```diff
@@ -15,19 +15,9 @@ def duration_display(minutes)
     hours = minutes / 60
     minutes = minutes - (hours * 60)
 
-    minute_string = minutes > 0 ? "#{minutes}m" : ""
-    hour_string = hours > 0 ? "#{hours}h" : ""
+    minute_string = minutes > 0 ? "#{minutes}" : ""
+    hour_string = hours > 0 ? "#{hours}" : ""
 
-    "#{hour_string} #{minute_string}".strip
-  end
-
-  def long_duration_display(minutes)
-    hours = minutes / 60
-    minutes = minutes - (hours * 60)
-
-    minute_string = minutes > 0 ? "#{minutes} #{'minute'.pluralize(minutes)}" : ""
-    hour_string = hours > 0 ? "#{hours} #{'hour'.pluralize(hours)}" : ""
-
-    "#{hour_string} #{minute_string}".strip
+    "#{hour_string}:#{minute_string}".strip
```

Comment:
@yock Thanks for those, this looks great!

I understand your concern about readability, and it's a concern that I share. RuboCop suggests using `hours.format('%02d')` over `'%02d' % hours` (likewise for minutes, of course). Does that seem like a reasonable readability improvement?